# HG changeset patch
# User rmunn
# Date 1506331983 -25200
#      Mon Sep 25 16:33:03 2017 +0700
# Branch 7000069
# Node ID 54e7f56337de7d4b1d59a2b76faf34d690e00e8e
# Parent  6512e8e194ff1e6474a6ca5de4a4e3e2e92c2f1c
[FLEx Bridge: 2.1] sync Comment about new word C

diff -r 6512e8e194ff -r 54e7f56337de Lexicon.fwstub.ChorusNotes
--- a/Lexicon.fwstub.ChorusNotes	Mon Sep 25 15:10:07 2017 +0700
+++ b/Lexicon.fwstub.ChorusNotes	Thu Dec 07 17:05:26 2017 +0100
@@ -21,4 +21,14 @@
 			date="2017-09-25T15:09:57Z"
 			guid="8f0d49e4-0a39-45b3-925d-bed838f968e1">Comment on word B</message>
 	</annotation>
+	<annotation
+		class="question"
+		ref="silfw://localhost/link?app=flex&amp;database=current&amp;server=&amp;tool=default&amp;guid=0ae86d59-9bfc-4704-b306-c488f37b2c43&amp;tag=&amp;id=0ae86d59-9bfc-4704-b306-c488f37b2c43&amp;label=C"
+		guid="274466cb-824a-4115-ab2f-385682fdc04f">
+		<message
+			author="rmunn"
+			status=""
+			date="2017-09-25T16:32:51Z"
+			guid="0b981639-50c0-4819-919c-94f0262c4cee">Comment about new word C</message>
+	</annotation>
 </notes>
\ No newline at end of file
diff -r 6512e8e194ff -r 54e7f56337de Linguistics/Lexicon/Lexicon_06.lexdb
--- a/Linguistics/Lexicon/Lexicon_06.lexdb	Mon Sep 25 15:10:07 2017 +0700
+++ b/Linguistics/Lexicon/Lexicon_06.lexdb	Thu Dec 07 17:05:26 2017 +0100
@@ -1,2 +1,49 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Lexicon />
\ No newline at end of file
+<Lexicon>
+	<LexEntry
+		guid="0ae86d59-9bfc-4704-b306-c488f37b2c43">
+		<DateCreated
+			val="2017-9-25 9:32:37.649" />
+		<DateModified
+			val="2017-9-25 9:32:37.649" />
+		<DoNotUseForParsing
+			val="False" />
+		<HomographNumber
+			val="0" />
+		<LexemeForm>
+			<MoStemAllomorph
+				guid="8ab74d13-0b7d-4b87-972a-731ca5851fde">
+				<Form>
+					<AUni
+						ws="fr">C</AUni>
+				</Form>
+				<IsAbstract
+					val="False" />
+				<MorphType>
+					<objsur
+						guid="d7f713e8-e8cf-11d3-9764-00c04f186933"
+						t="r" />
+				</MorphType>
+			</MoStemAllomorph>
+		</LexemeForm>
+		<MorphoSyntaxAnalyses>
+			<MoStemMsa
+				guid="835c6d05-665c-4ac4-9aa2-fe50db9cedbb" />
+		</MorphoSyntaxAnalyses>
+		<Senses>
+			<ownseq
+				class="LexSense"
+				guid="6f41e434-a753-4ece-a3b5-6d2c6aa36314">
+				<Gloss>
+					<AUni
+						ws="en">C</AUni>
+				</Gloss>
+				<MorphoSyntaxAnalysis>
+					<objsur
+						guid="835c6d05-665c-4ac4-9aa2-fe50db9cedbb"
+						t="r" />
+				</MorphoSyntaxAnalysis>
+			</ownseq>
+		</Senses>
+	</LexEntry>
+</Lexicon>
